shared/profile/edit-avatar/index.native.tsx

```diff
@@ -234,7 +233,7 @@ class AvatarUpload extends React.Component<Props & WrappedProps> {
                 : Styles.collapseStyles([
                     styles.zoomContainer,
                     {
-                      borderRadius: this.props.type === 'team' ? 32 : this.avatar_size(),
```

Comment:
This seems like it makes teams circles which doesn't seem like what you want?

Reply:
at this point `this.props.type` is always 'profile' given we're checking for 'team' on line 166 (we're in the `else` case here basically). Before when we had that flag you could get here if `teamsRedesign` was false but now we're always in the `'profile'` state. so if it is incorrect, it was incorrect before